data_safe_haven/infrastructure/components/composite/postgresql_database.py

```diff
@@ -81,9 +81,11 @@ def __init__(
             f"{self._name}_secure_transport_configuration",
             configuration_name="require_secure_transport",
             resource_group_name=props.database_resource_group_name,
-            server_name=props.database_server_name,
+            server_name=db_server.name,
             source="user-override",
             value="OFF",
+            opts=ResourceOptions.merge(child_opts, ResourceOptions(parent=db_server)),
+            tags=child_tags,
```

Comment:
I don't think this resource has tags - it's not a thing that exists on its own in the portal, it's a setting on an existing resource.